hnn_core/tests/test_gui.py

```diff
@@ -1116,3 +1116,53 @@ def test_delete_single_drive(setup_gui):
                                           'alpha_prox (proximal)',
                                           'poisson (proximal)',
                                           'tonic')
+
+
+def test_default_smoothing(setup_gui):
+    """Tests default smoothing is inherited correctly"""
+
+    gui = setup_gui
+    gui.run_button.click()
+
+    gui.widget_simulation_name.value = 'no_smoothing'
+
+    # change value of default smoothing via the widget
+    new_smoothing = 0
+    gui.widget_default_smoothing.value = new_smoothing
+
+    gui.run_button.click()
+
+    # check that new default smoothing value is set everywhere
+    gui_smooth_value = gui.fig_default_params['default_smoothing']
+    viz_smooth_value = gui.viz_manager.fig_default_params['default_smoothing']
+
+    assert gui_smooth_value == new_smoothing
```

Comment:
It'd be useful to add a similar assert for the original smooth values of the `fig_default` and `viz_manager`